tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -460,11 +459,19 @@ def test_validate_nonce_and_state():
 
 
 @pytest.mark.django_db
-def test_generate_client_assertion():
+def test_generate_client_assertion_base64():
     """Test client assertion generation."""
     os.environ["JWT_KEY"] = test_private_key
     assert generate_client_assertion() is not None
 
+@pytest.mark.django_db
+def test_generate_client_assertion_pem():
+    """Test client assertion generation."""
+    from base64 import b64decode
+    os.environ["JWT_KEY"] = b64decode(test_private_key).decode("utf-8")
+    utf8_jwt_key = generate_client_assertion()
+    assert utf8_jwt_key is not None
```

Comment:
Because the response is an encrypted JWT we really don't have a way to predict what the output is, but `not None` is a valid test because if there was no response it would be an error, which would fail the test - not to mention that all of our `auth` tests that use the key would also fail, like we saw with the problem yesterday.

Reply:
Ah I see, and the encrypted key includes (a) a random number and (b) a value based on `time.now`. 

Thinking out loud ... we could assert that the key is a string, and I bet we could assert the length of the string is a specific value, does that seem right to you?

Reply:
Yes, we could assert that it's a string. I also had the thought on the length, but looking into it further I found that there is no set length for a JWT string, so that part wouldn't be testable.